src/sora-academy.md

```diff
@@ -1,3 +1,53 @@
+
+Through swapping and pooling, different tokens can increase their market value. When you swap, you purchase tokens using a base token of your choice, this could be considered as the digital version of a forex exchange, where you trade one currency for another based on the exchange rate. This is directly proportional to pooling, where someone holding an amount of both currencies deposits an equivalent amount of both in a pool, where other users have tokens deposited too, and whenever someone makes a swap of the currencies in a pool you belong to, you receive a commission proportional to the amount of liquidity you have provided.
+
+A different way to explain this would be the commission that the forex bureau keeps from the total amount of money you have exchanged. In the case of DeFi, the commission goes to more people, and the price of either currency (token) is defined by the amount of liquidity in the pool.
+
+Last but not least, NFT trading and collecting is one of the newer forms of DeFi, there are items that have made headlines due to the exorbitant value they fetch when sold, such as the [CryptoPunk #7523](https://news.artnet.com/market/sothebys-natively-digital-sale-1979174), sold in a Sotheby’s auction for millions of dollars. Before you ask, yes, this is the same Sotheby’s that sells Picasso and Van Gogh art.
+
+How a pixelated image of a humanoid figure can cost so much is in part because it was one of the first NFT ever made (minted) on the Ethereum blockchain, but also due to the same nuances that make Cabbage Patch Kids dolls or Pokémon trading cards collector’s items. However, it is important to note that NFTs are not only images, but there are also many items that can be made into NFT, such as videos, songs, and even items to be used in games. The options are infinite (and so is the value they can fetch).
+
+At this point, you might be wondering where are all these transactions registered? It does seem very abstract that each platform can interact with their underlying smart contracts, but in your wallet, the transactions aren’t specifying what is going on. But blockchains are supposed to foster transparency, right? That is correct. The tool used to have an overview of all the transactions, ranging from transferring tokens from one account to another or the most complicated DeFi operations can all be audited using a block explorer. These tools allow you to review the transactions contained within each block of the chain. It is important to note that there are different explorers for each blockchain, and some have more than one explorer.
```

Comment:
> The tool used to have an overview of all the transactions, ranging from transferring tokens from one account to another or the most complicated DeFi operations can all be audited using a block explorer.

this sentence is rather hard to comprehend

Reply:
Open to suggestions on how to simplify :D

Reply:
maybe something like this 

```suggestion
At this point, you might be wondering where are all these transactions registered? It does seem very abstract that each platform can interact with their underlying smart contracts, but in your wallet, the transactions aren’t specifying what is going on. But blockchains are supposed to foster transparency, right? That is where block explorers come in. A block explorer allows you to review transactions within each block of the chain. You can audit a wide range of operations, from transferring tokens from one account to another to the most complicated DeFi operations. It is important to note that there are different explorers for each blockchain, and some have more than one explorer.
```
>